cl_dll/hud_countdown.cpp

```diff
@@ -19,6 +19,7 @@ int CHudCountdown::Init()
 	name2[ARRAYSIZE(name2) - 1] = '\0';
 
 	cl_autorecord = CVAR_CREATE("cl_autorecord", "0", FCVAR_ARCHIVE);
+	cl_autorecord_stop = CVAR_CREATE("cl_autorecord_stop", "1", FCVAR_ARCHIVE);
```

Comment:
Don't default this to enabled as it changes existing behavior.
```suggestion
	cl_autorecord_stop = CVAR_CREATE("cl_autorecord_stop", "0", FCVAR_ARCHIVE);
```